### DISCUSSION TITLE
Should this be a monorepo?

### DISCUSSION THREAD

**Discussion by jvanulde** [Canadian-Geospatial-Platform/geo.ca#62] [Ideas]
@PVautour @indiciumx should this be a monorepo? If so, what repos should/could be included?

**Comment by pvautour** [comment]
We could maybe have two repos?

1. Repo containing static release files, sitemaps, etc.
2. Repo containing cloudformations for AWS backend code and cloudformations.

Having one big repo would certainly help with getting an understanding of the project as a whole, but stuffing content and code together could create a pretty bloated repo. Especially if we dont have a special system to handle binary files. The content repo will probably need to be culled often.

Keeping content in it's own repo and code/infra in its own will also keep commit history cleaner/more meaningful.

What do you think?

If not we could leverage submodules, but that adds extra tooling and complexity for unclear gains.
